src/__tests__/settings/Settings.test.tsx

```diff
@@ -0,0 +1,34 @@
+import '../../__mocks__/matchMediaMock'
+
+import { mount } from 'enzyme'
+import { createMemoryHistory } from 'history'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Router } from 'react-router-dom'
+import createMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+import * as titleUtil from '../../page-header/useTitle'
+import Settings from '../../settings/Settings'
+import { RootState } from '../../store'
+
+const mockStore = createMockStore<RootState, any>([thunk])
+
+describe('Settings', () => {
+  const store = mockStore({ title: 'test' } as any)
+  const history = createMemoryHistory()
+  const titleSpy: any = jest.spyOn(titleUtil, 'default')
+  history.push('/settings')
+
+  mount(
+    <Provider store={store}>
+      <Router history={history}>
+        <Settings />
+      </Router>
+    </Provider>,
+  )
```

Comment:
let's move this to a setup method that would look something like this: 

```ts
const setup = () => {
	  const history = createMemoryHistory()
	  jest.spyOn(titleUtil, 'default')
	  history.push('/settings')
	  const wrapper = mount(
	    <Provider store={store}>
	      <Router history={history}>
	        <Settings />
	      </Router>
	    </Provider>,
	  )

	return { wrapper }
}
```

Then the test would look like: 

```ts
  it('should have Settings as the title', () => {
	setup();
    expect(titleUtil.default).toHaveBeenCalledWith('settings.label')
  })
```

Sorry to be super nitpicky, however, this will help us as the component grows.

Reply:
100% and so clean. ✅ in 08e08e6